Dockerfile.ocp

```diff
@@ -24,6 +24,7 @@ LABEL url="https://access.redhat.com/"
 LABEL vendor="Red Hat, Inc."
 LABEL version="1"
 LABEL maintainer="Red Hat"
+LABEL io.openshift.tags="instaslice"
```

Comment:
We should use official name here.

Reply:
I wasn't sure what to put  here.  It is just a list of key words.  This was just a placeholder to make sure the EC violations went away.  I can put whatever words we want here.

Reply:
these are list of tags:

```
das,accelerator,dynamic,instaslice,slicer
```